CMakeLists.txt

```diff
@@ -278,11 +278,17 @@ set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}")
 endif()
 
 ### ---[ Find universal dependencies
-find_package(OpenMP)
+find_package(OpenMP COMPONENTS C CXX)
 if(OPENMP_FOUND)
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-  message(STATUS "Found OpenMP")
+  if(${CMAKE_VERSION} VERSION_LESS "3.7")
+    message(STATUS "Found OpenMP")
+  else()
+    # We could use OpenMP_CXX_VERSION starting from CMake 3.9, but this value is not set, even it should,
```

Comment:
3.9 or 3.7?

Reply:
I will clarify this comment with my next commit (`OpenMP_CXX_VERSION` is available since CMake 3.9, but `OpenMP_CXX_SPEC_DATE` since CMake 3.7).